src/components/navigation/www/navigation.jsx

```diff
@@ -77,6 +69,23 @@ class Navigation extends React.Component {
         if (!this.props.user) return;
         return `/users/${this.props.user.username}/`;
     }
+
+    setupMessagePolling (minutes) {
+        this.props.getMessageCount(this.props.user.username);
+        // We only poll if it has been less than 30 minutes.
+        // Chances of someone actively using the page for that long without
+        // a navigation is low.
+        if (minutes < 32) {
+            const nextFetch = minutes * 2;
+            const timeoutId = setTimeout(() => {
+                this.setupMessagePolling(nextFetch);
+            }, nextFetch * 60000);
```

Comment:
The convention you used earlier, on line 47, where you showed the multiplication that got you this number ("2 * 60 * 1000") might also be good here

Reply:
I changed everything over to being in ms as you suggested. Since I did that, nextFetch is now in millis so I don't need the multiplication anymore.